projects/m1/024-fizz-buzz/js/index.js

```diff
@@ -0,0 +1,34 @@
+// const prompt = require("prompt-sync")({ sigint: true}) ;
+
+function playFizzBuzz(){
+    for (let i = 1; i <= 100; i++) {
+        let phrase = i;
+
+        const divisbleBy3 = i % 3 == 0 ? true: false;
+        const divisbleBy5 = i % 5 == 0 ? true: false;
+
+        if ( divisbleBy3 ){
+            phrase = 'fizz';
+        }
+
+        if ( divisbleBy5 ){
+            phrase = 'buzz';
+        }
+
+        if ( divisbleBy3 && divisbleBy5 ){
+            phrase = 'fizz and buzz';
+        }
+
+        console.log( phrase );
+    }
+
+    return;
+    
+}
+
+function init(){
+    playFizzBuzz();
+    return;
+}
+
+init();
```

Comment:
In general, I advise **not to complicate the solution** and to solve it with at most three conditions.

Reply:
Hi,
can you kindly check that everything is ok now and resolve the pr?

Thank you.

Reply:
This algorithm does not solve the problem correctly.
But at this point, I suggest you not change the code now.
Write the tests first and see if it really works as you would expect or not.
In short, correct it after testing it ;-)